docker/scan-cli-plugin
# fix issue when scanning a local image with the snyk containerized pro…

…vider

Signed-off-by: Guillaume Lours <guillaume.lours@docker.com>

fixes #151 

**- What I did**
Use the `snyk/snyk:docker` image instead of the `snyk/snyk:alpine` one

**- How I did it**
Change the SHA of the image in the `vars.mk` file
Add a e2e test which build a local image and try to scan this image with the containerized Snyk Provider

**- How to verify it**
e2e test should pass 
On linux : 
```
$ docker build -f ./e2e/testdata/Dockerfile -t local:build .

# when the image have been built 
$ docker scan local:build
...
Organization:      docker-desktop-test
Package manager:   apk
Project name:      docker-image|local
Docker image:      local:build
Platform:          linux/amd64
Licenses:          enabled

Tested 14 dependencies for known issues, found 15 issues.
```

**- A picture of a cute animal (not mandatory)**

![image](https://user-images.githubusercontent.com/705411/115743752-3bdfaf80-a392-11eb-9514-52059103145d.png)
